apps/examples/testcase/le_tc/filesystem/fs_main.c

```diff
@@ -2165,16 +2225,44 @@ static void libc_stdio_memsistream_tc(void)
 */
 static void libc_stdio_memsostream_tc(void)
 {
```

Comment:
All TCs for outstream look wrong because you write something to same memory.
you set buffer for stream and use it for put function again.

Reply:
@jeongarmy please check now